src/sentry/snuba/models.py

```diff
@@ -41,6 +41,31 @@ class Meta:
         app_label = "sentry"
         db_table = "sentry_snubaquery"
 
+    @property
+    def event_types(self):
+        return [type.event_type for type in self.snubaqueryeventtype_set.all()]
+
+
+class SnubaQueryEventType(Model):
+    __core__ = True
+
+    class EventType(Enum):
+        ERROR = 0
+        DEFAULT = 1
```

Comment:
What about CSP and security type events?

Reply:
We don't care about these for the moment, although I can include them here for completeness if you prefer.